### PR TITLE
fix app stop processing if no printer detected 

### PR DESCRIPTION
**Pull request by ahmadsoran**
in this PR i fixed an issue happened to me with USB adapter pkg if no usb printer detected will stop electron app from processing & added paper cut command support specific type of thermal printers

### CODE REVIEW

File: packages/usb-adapter/src/index.ts [hunk 1]
```diff
@@ -47,9 +47,12 @@ export default class USBAdapter extends Adapter<[]> {
         this.device = devices[0];
     }
     if (!this.device)
-      throw new Error("Can not find printer");
-
-    usb.on("detach", this.detachDevice);
+    // throw new Error("Can not find printer") will cause process exit due to uncaught exception.
+    // console if device not found 
+    console.error("Printer not found maybe not connected or not installed yet !");
```
Comment (dohooo) [inline packages/usb-adapter/src/index.ts:52]: Why don't you catch errors externally?

File: packages/core/src/commands.ts [hunk 2]
```diff
@@ -82,13 +82,16 @@ export const MARGINS = {
  * [PAPER Paper]
  * @type {Object}
  */
-export const PAPER = {
+
```
Comment (dohooo) [inline packages/core/src/commands.ts:85]: Extra spacer.

**Comment by dohooo** [comment]
IMHO, this change seems unnecessary. Can you show me your code?

**Comment by ahmadsoran** [comment]
seems you'r right thx,
 what i was thinking :D

**Pull request closed without merging**